lib/commands/prop/prop-set-android-version-code.ts

```diff
@@ -0,0 +1,27 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import {SetProjectPropertyCommand} from "./prop-set";
+
+export class SetAndroidVersionCodeCommand extends SetProjectPropertyCommand implements ICommand {
+	constructor($staticConfig: IStaticConfig,
+		$injector: IInjector,
+		private $logger: ILogger,
+		private $projectConstants: Project.IProjectConstants) {
+		super($staticConfig, $injector);
+	}
+
+	canExecute(args: string[]): IFuture<boolean> {
+		return this.$project.validateProjectProperty("AndroidVersionCode", args, "set");
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return ((): void => {
+			super.execute(["AndroidVersionCode", args[0]]).wait();
+			if (this.$project.projectData.Framework === this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova) {
+				this.$logger.printMarkdown("Please note that the actual AndroidVersionCode will be `%s2` due to an implementation decision in Apache Cordova.", args[0]);
```

Comment:
Your final AndroidVersionCode will be `%s2` because Apache Cordova automatically appends a specific number to the version code based on the target Android SDK and architecture. For more information, see https://issues.apache.org/jira/browse/CB-8976.